spec/00-abstract.md

```diff
@@ -0,0 +1,6 @@
+## Abstract
+
+This specification defines a standard format to propagate context
+information for distributed traces. The specification standardizes how context
```

Comment:
```suggestion
information for distributed traces. Context information identifies individual requests in a distributed system and also provides means to add and propagate information that is specific for a tracing vendor.
```

Reply:
this comment tell explicitly that this spec is a first spec the identifies individual request

Reply:
I believe it's important to note that this is the first spec that defines request identification mechanism.

Reply:
Pushed a commit that resolves it